lib/generate.js

```diff
@@ -135,6 +135,33 @@ generate.javascript.builtinPostprocessors = {
     "id": "id"
 }
 
+
+generate.es2015 = generate.es6 = function (parser, exportName) {
+    return "// Generated automatically by nearley\n" +
+        "// http://github.com/Hardmath123/nearley\n" +
+        "{\n" +
+        "const id = x => x[0];\n" +
+        parser.body.join('\n') +
+        "const grammar = {\n" +
+        "    Lexer: " + parser.config.lexer + ",\n" +
+        "    ParserRules: " + serializeRules(parser.rules, generate.es2015.builtinPostprocessors) + ",\n" +
+        "    ParserStart: " + JSON.stringify(parser.start) + "\n" +
+        "}\n" +
```

Comment:
Heh. Side effect of copying from the regular JS generator (see line 120).

If it's all right with you, I'd love to make another PR to make the coding style consistent.